packaging/ubuntu-14.04/control

```diff
@@ -118,3 +118,11 @@ Section: oldlibs
 Pre-Depends: dpkg (>= 1.15.7.2)
 Description: Transitional package for snap-confine
  This is a transitional dummy package. It can safely be removed.
+
+Package: snapd-xdg-open
+Architecture: any
+Depends: snap-confine (= ${binary:Version}), ${misc:Depends}
```

Comment:
I guess this just me doing a copy&paste of the previous entry in the control file. Let me drop that.